integration-tests/environments/electron/jenkins.groovy

```diff
@@ -30,36 +30,40 @@ def onLinux() {
       ) {
         // Unstash the package produced when packaging
         dir('integration-tests') {
+          // Remove any archive from the workspace, which might have been produced by previous runs of the job
+          sh 'rm -f realm-*.tgz'
           unstash 'package'
         }
         // Install the packaged version of realm into the app and run the tests
         dir('integration-tests/environments/electron') {
           // Install the package, leaving out the optional packages to prevent Realm being installed from NPM
           sh 'npm install --no-optional'
-          // Run both main and renderer tests catching any errors
-          def error = null;
-          // First the main process
-          try {
-            // Using xvfb to allow Electron to open a window
-            sh 'xvfb-run npm run test/main -- main-test-results.xml'
-          } catch (Exception e) {
-            error = e;
-          }
-          // Then the renderer process
-          try {
-            // Using xvfb to allow Electron to open a window
-            sh 'xvfb-run npm run test/renderer -- renderer-test-results.xml'
-          } catch (Exception e) {
-            error = e;
-          }
-          // Archive all test results
-          junit(
-            allowEmptyResults: true,
-            testResults: '*-test-results.xml',
-          )
-          // Throw any errors that might have occurred
-          if (error) {
-            throw error
+          timeout(10) { // minutes
```

Comment:
When we add more tests, I think that 10 minutes is a very low value.

Reply:
I've increased it to 30 minutes - it shouldn't take longer to run a test.